### PR TITLE
Fix feedstock

### LINKED ISSUE #6: error with bfg script
Issue: bfg script does not use correct CONDA_PREFIX

When using the bfg executable that is generated in recipe/build.sh, you get:

```
$ bfg
Error: Unable to access jarfile /lib/bfg.jar
````

In build.sh, 
```
echo "java -jar ${PREFIX}/lib/bfg.jar \"$@\"" >> $PREFIX/bin/bfg
```
should be 
```
echo "java -jar ${CONDA_PREFIX}/lib/bfg.jar \"$@\"" >> $PREFIX/bin/bfg
```
Then it works. 

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

$ conda list
# packages in environment at /data/rajewsky/home/dkoppst/miniconda3:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       0_gnu    conda-forge
alsa-lib                  1.1.5             h516909a_1002    conda-forge
bcl2fastq                 2.20                          0    freenome
bfg                       1.13.0                        1    conda-forge
ca-certificates           2019.11.28           hecc5488_0    conda-forge
certifi                   2019.11.28               py36_0    conda-forge
cffi                      1.13.2           py36h8022711_0    conda-forge
chardet                   3.0.4                 py36_1003    conda-forge
conda                     4.8.2                    py36_0    conda-forge
conda-package-handling    1.6.0            py36h516909a_1    conda-forge
cryptography              2.8              py36h72c5cf5_1    conda-forge
curl                      7.68.0               hf8cf82a_0    conda-forge
expat                     2.2.9                he1b5a44_2    conda-forge
fontconfig                2.13.1            h86ecdb6_1001    conda-forge
freetype                  2.10.0               he983fc9_1    conda-forge
gettext                   0.19.8.1          hc5be6a0_1002    conda-forge
giflib                    5.2.1                h516909a_2    conda-forge
git                       2.23.0          pl526hce37bd2_2    conda-forge
icu                       64.2                 he1b5a44_1    conda-forge
idna                      2.8                   py36_1000    conda-forge
jpeg                      9c                h14c3975_1001    conda-forge
krb5                      1.16.4               h2fd8d38_0    conda-forge
lcms2                     2.9                  hbd6801e_1    conda-forge
libcurl                   7.68.0               hda55be3_0    conda-forge
libedit                   3.1.20170329      hf8c457e_1001    conda-forge
libffi                    3.2.1             he1b5a44_1006    conda-forge
libgcc-ng                 9.2.0                h24d8f2e_2    conda-forge
libgomp                   9.2.0                h24d8f2e_2    conda-forge
libiconv                  1.15              h516909a_1005    conda-forge
libpng                    1.6.37               hed695b0_0    conda-forge
libssh2                   1.8.2                h22169c7_2    conda-forge
libstdcxx-ng              9.2.0                hdf63c60_2    conda-forge
libtiff                   4.1.0                hc3755c2_3    conda-forge
libuuid                   2.32.1            h14c3975_1000    conda-forge
libxcb                    1.13              h14c3975_1002    conda-forge
libxml2                   2.9.10               hee79883_0    conda-forge
lz4-c                     1.8.3             he1b5a44_1001    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
openjdk                   11.0.1            h600c080_1018    conda-forge
openssl                   1.1.1d               h516909a_0    conda-forge
pcre                      8.43                 he1b5a44_0    conda-forge
perl                      5.26.2            h516909a_1006    conda-forge
pip                       20.0.2                     py_2    conda-forge
pthread-stubs             0.4               h14c3975_1001    conda-forge
pycosat                   0.6.3           py36h516909a_1002    conda-forge
pycparser                 2.19                     py36_1    conda-forge
pyopenssl                 19.1.0                   py36_0    conda-forge
pysocks                   1.7.1                    py36_0    conda-forge
python                    3.6.7             h357f687_1005    conda-forge
readline                  8.0                  hf8c457e_0    conda-forge
requests                  2.22.0                   py36_1    conda-forge
ruamel_yaml               0.15.80         py36h516909a_1000    conda-forge
setuptools                45.2.0                   py36_0    conda-forge
six                       1.14.0                   py36_0    conda-forge
sqlite                    3.30.1               hcee41ef_0    conda-forge
tk                        8.6.10               hed695b0_0    conda-forge
tqdm                      4.7.2                    py36_0    bioconda
urllib3                   1.25.7                   py36_0    conda-forge
wheel                     0.34.2                     py_1    conda-forge
xorg-fixesproto           5.0               h14c3975_1002    conda-forge
xorg-inputproto           2.3.2             h14c3975_1002    conda-forge
xorg-kbproto              1.0.7             h14c3975_1002    conda-forge
xorg-libx11               1.6.9                h516909a_0    conda-forge
xorg-libxau               1.0.9                h14c3975_0    conda-forge
xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
xorg-libxext              1.3.4                h516909a_0    conda-forge
xorg-libxfixes            5.0.3             h516909a_1004    conda-forge
xorg-libxi                1.7.10               h516909a_0    conda-forge
xorg-libxrender           0.9.10            h516909a_1002    conda-forge
xorg-libxtst              1.2.3             h516909a_1002    conda-forge
xorg-recordproto          1.14.2            h516909a_1002    conda-forge
xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
xorg-xextproto            7.3.0             h14c3975_1002    conda-forge
xorg-xproto               7.0.31            h14c3975_1007    conda-forge
xz                        5.2.4             h14c3975_1001    conda-forge
yaml                      0.2.2                h516909a_1    conda-forge
zlib                      1.2.11            h516909a_1006    conda-forge
zsh                       5.2                           0    activisiongamescience
zstd                      1.4.4                h3b9ef0a_1    conda-forge

```

```
(base) [dkoppst@murphy:/data/rajewsky/home/dkoppst] (1029) $ conda info

     active environment : base
    active env location : /data/rajewsky/home/dkoppst/miniconda3
            shell level : 1
       user config file : /home/dkoppst/.condarc
 populated config files : /home/dkoppst/.condarc
          conda version : 4.8.2
    conda-build version : not installed
         python version : 3.6.7.final.0
       virtual packages : __glibc=2.17
       base environment : /data/rajewsky/home/dkoppst/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/bioconda/linux-64
                          https://conda.anaconda.org/bioconda/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /data/rajewsky/home/dkoppst/miniconda3/pkgs
                          /home/dkoppst/.conda/pkgs
       envs directories : /data/rajewsky/home/dkoppst/miniconda3/envs
                          /home/dkoppst/.conda/envs
               platform : linux-64
             user-agent : conda/4.8.2 requests/2.22.0 CPython/3.6.7 Linux/3.10.0-1062.9.1.el7.x86_64 centos/7.7.1908 glibc/2.17
                UID:GID : 24807:24807
             netrc file : None
           offline mode : False


```
</details>

### PR DESCRIPTION
**Pull request by janjagusch**
Closes #6 

This feedstock hasn't received some love in a while. I fixed a couple of things and now it's working again.

- The source now gets downloaded using `https` (otherwise the build would fail)
- The `/bin/bfg` script now contains the correct prefix to find the `bfg.jar`
- I removed a maintainer whose GitHub account doesn't exist anymore and added myself as a maintainer. I hope that's okay
- Fixed the SPDX identifier of the license file
- Added a more meaningful test, where I check that the correct version is installed

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by janjagusch** [comment]
@conda-forge-admin, please rerender

File: recipe/bld.bat [hunk 1]
```diff
@@ -1,4 +1,4 @@
 mkdir "%PREFIX%"\lib
 mkdir "%SCRIPTS%"\
 copy "%SRC_DIR%\%PKG_NAME%-%PKG_VERSION%.jar" "%PREFIX%\lib\bfg.jar"
-echo java -jar %PREFIX%\lib\bfg.jar %%* >> "%SCRIPTS%\bfg.bat"
+echo java -jar ^%CONDA_PREFIX^%\lib\bfg.jar %%* >> "%SCRIPTS%\bfg.bat"
```
Comment (xhochy) [inline recipe/bld.bat]: If you leave `%PREFIX%` here, conda should automatically replace this with the correct path on installation.

**Comment by carlodri** [comment]
@janjagusch AFAIR windows build was working fine, it was Linux that had some issues.

**Comment by janjagusch** [comment]
@carlodri, I have reverted the windows build to its version on the default branch and have fixed the windows test (turns out an excess `}` was causing problems). All tests are passing now. 👍 

Anything else I should take a look at before we can get this merged?

**Comment by carlodri** [comment]
Thanks for your work on this @janjagusch!

**Pull request merged**